eng/pipelines/sync-sharedcode.yml

```diff
@@ -0,0 +1,47 @@
+trigger: 'none'
+pr: 'none'
+
+schedules:
+- cron: '0 0 * * *'
+  displayName: 'Daily Synchronize Azure Core Shared Codes'
+  branches:
+    include:
+    - feature/v3
+  always: true
+
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+      ref: refs/tags/azure-sdk-tools_20220404.3
+
+stages:
+  - stage: Synchronize_Shared_Codes
+    jobs:
+       - job: Download_And_Create_PR
+         pool:
+            vmImage: ubuntu-20.04
+         variables:
+            currentDateTime: $[ format('{0:yyyy}-{0:MM}-{0:dd}-{0:HH}-{0:mm}-{0:ss}', pipeline.startTime) ]
```

Comment:
`2023-02-16-19-55-06` is not very readable. Can we use a more readbale format like `2023-02-16_19:55:06`?